Doc/Makefile

```diff
@@ -40,7 +40,19 @@ help:
 
 build:
 	-mkdir -p build
-	$(BLURB) merge -f build/NEWS
+	# Look first for a Misc/NEWS file (building from a source release tarball
+	# or old repo) and use that, otherwise look for a Misc/NEWS.d directory
+	# (building from a newer repo) and use blurb to generate the NEWS file.
+	@if [ -f  ../Misc/NEWS ] ; then \
+		echo "Using existing Misc/NEWS file"; \
+		cp ../Misc/NEWS build/NEWS; \
+	elif [ -d ../Misc/NEWS.d ]; then \
+		echo "Building NEWS from Misc/NEWS.d with blurb"; \
+		$(BLURB) merge -f build/NEWS; \
+	else \
+		echo "Neither Misc/NEWS.d nor Misc/NEWS found; cannot build docs"; \
```

Comment:
Yeah, we could make that non-fatal.  But one or the other (Misc/NEWS or Misc/NEWS.d should be there, I'd think.  And the previous Docs build did assume that Misc/NEWS was in the parent directory so that's not new; it might be nice to allow for out of tree builds but that's a separate feature.